src/main/java/org/folio/inventory/support/ItemUtil.java

```diff
@@ -26,10 +29,155 @@
   private static final String TEMPORARY_LOAN_TYPE_ID_KEY = "temporaryLoanTypeId";
   private static final String PERMANENT_LOCATION_ID_KEY = "permanentLocationId";
   private static final String TEMPORARY_LOCATION_ID_KEY = "temporaryLocationId";
+  public static final String HOLDINGS_RECORD_ID = "holdingsRecordId";
+  public static final String STATUS = "status";
+  public static final String NUMBER_OF_PIECES = "numberOfPieces";
+  public static final String COPY_NUMBER = "copyNumber";
+  public static final String CHRONOLOGY = "chronology";
+  public static final String ENUMERATION = "enumeration";
+  public static final String BARCODE = "barcode";
+  public static final String NOTES = "notes";
+  public static final String ID = "id";
+  public static final String MATERIAL_TYPE = "materialType";
+  public static final String PERMANENT_LOCATION = "permanentLocation";
+  public static final String TEMPORARY_LOCATION = "temporaryLocation";
+  public static final String PERMANENT_LOAN_TYPE = "permanentLoanType";
+  public static final String TEMPORARY_LOAN_TYPE = "temporaryLoanType";
 
   private ItemUtil() {
   }
 
+  public static Item jsonToServerItem(JsonObject itemFromServer) {
```

Comment:
I'd use `storedItem` here, what do you think? 

```suggestion
  public static Item storedItemJsonToItem(JsonObject itemFromServer) {
```

Reply:
Or
```suggestion
  public static Item fromStoredItemRepresentation(JsonObject itemFromServer) {
```